raspbian/updating.md

```diff
@@ -64,7 +64,7 @@ If moving to a new Pi model (for example the Pi 3B+), you may also need to updat
 
 ## Third-party solutions
 
-This section addresses why third-party solutions are of interest and why [apt](../linux/software/apt.md) is not optimal for all situations. It also covers existing third-party solutions that support Raspbian.
+This section addresses why third-party solutions are of interest and why [apt](../linux/software/apt.md) is not optimal for all situations. Raspberry Pi do not recommend any specific third party tools, prospective users should determine the most suitable tool for their particular requirements.
```

Comment:
I think I'd be tempted to change "why third-party solutions are of interest" to "why third-party solutions may be of interest" ?  And "third party" in the second sentence should probably be hyphenated?

Reply:
Good catch - 'may be' would be a useful addition.

> And "third party" in the second sentence should probably be hyphenated?

I thought so too, but according to the Oxford English Dictionary (see https://www.lexico.com/definition/third_party) the correct spelling is as-is, i.e. 'third party' without the hyphen.

Reply:
That page says that it **is** hyphenated when used as an adjective rather than a noun? :stuck_out_tongue_winking_eye:

Reply:
> That page says that it **is** hyphenated when used as an adjective rather than a noun? 😜

Ooops - sorry - I didn't read the page fully. My bad.

Reply:
Added hyphen, or as I like to say, subtraction sign.